hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/transfer/CustomFeeAssessmentStepTest.java

```diff
@@ -101,6 +99,40 @@ void hbarFixedFeeAndRoyaltyFeeWithFallback() {
         assertThatTransfersContains(givenOp.transfers().accountAmounts(), expectedGivenOpHbarTransfers);
     }
 
+    @Test
+    @DisplayName("Transfer which adds a custom fee to the metadata")
+    void hbarFixedCustomFee() {
```

Comment:
Wasn't this test part of a previous PR?

Reply:
Yes, I'm not sure why that's in there. Probably because the previous PR hasn't been merged yet.